packages/office-ui-fabric-react/src/components/ChoiceGroupOption/ChoiceGroupOption.base.tsx

```diff
@@ -0,0 +1,169 @@
+import * as React from 'react';
+import { Image } from '../../Image';
+import { Icon } from '../../Icon';
+import { IClassNames } from '@uifabric/utilities/lib/IClassNames';
+import {
+  IChoiceGroupOptionProps,
+  IChoiceGroupOptionStyleProps,
+  IChoiceGroupOptionStyles
+} from './ChoiceGroupOption.types';
+import {
+  BaseComponent,
+  customizable,
+  classNamesFunction,
+  getNativeProps,
+  inputProperties,
+  createRef,
+} from '../../Utilities';
+
+const getClassNames = classNamesFunction<IChoiceGroupOptionStyleProps, IChoiceGroupOptionStyles>();
+
+@customizable('ChoiceGroupOption', ['theme'])
+export class ChoiceGroupOptionBase extends BaseComponent<IChoiceGroupOptionProps, any> {
+  private _inputElement = createRef<HTMLInputElement>();
+  private _classNames: IClassNames<IChoiceGroupOptionStyles>;
+
+  constructor(props: IChoiceGroupOptionProps) {
+    super(props);
+  }
+
+  public render(): JSX.Element {
+    const {
+      focused,
+      required,
+      theme,
+      iconProps,
+      imageSrc,
+      imageSize,
+      disabled,
+      checked,
+      id,
+      labelId,
+      getStyles,
+      name,
+      onRenderField = this._onRenderField,
+    } = this.props;
+
+    this._classNames = getClassNames(getStyles!, {
+      theme: theme!,
+      hasIcon: !!iconProps,
+      hasImage: !!imageSrc,
+      checked,
+      disabled,
+      imageIsLarge: !!imageSrc && (imageSize!.width > 71 || imageSize!.height > 71),
+      focused
+    });
+
+    return (
+      <div className={ this._classNames.root }>
+        <div className={ this._classNames.choiceFieldWrapper }>
+          <input
+            { ...getNativeProps(this.props, inputProperties) }
```

Comment:
My understanding here is that stomping over getNativeProps is undesirable and the spread should really be at the end of the <input> tag.  I ran into issues with that ordering, but I want to leave this comment here to solicit commentary and to remind myself to revisit whether or not I'm still seeing issues with this ordering.